google/cloud/documentai_toolbox/utilities/docai_utilities.py

```diff
@@ -37,8 +37,8 @@ def get_bounding_box(
             Bounding box coordinates in order (top, left, bottom, right).
             Returns `0, 0, 0, 0` if `bounding_poly.normalized_vertices` is empty.
     """
-    if not bounding_poly.normalized_vertices:
-        return 0, 0, 0, 0
+    if not bounding_poly or not bounding_poly.normalized_vertices:
```

Comment:
Update the docstring for this changed behavior.

Reply:
Missed this.